tests/unit/test_base_command.py

```diff
@@ -91,67 +91,51 @@ def test_handle_pip_version_check_called(mock_handle_version_check):
     mock_handle_version_check.assert_called_once()
 
 
-class Test_base_command_logging(object):
-    """
-    Test `pip.base_command.Command` setting up logging consumers based on
-    options
-    """
+@pytest.fixture
+def utc():
```

Comment:
This could be a `fixed_time` fixture that would also use a shared `utc` fixture with the one defined in ` tests/unit/test_logging.py`

Reply:
In cd549eb7f1f8de518929381de698bfa32c33ba0d I moved this `utc` fixture to the test suite's root conftest.  I hope that's OK.

Reply:
Thanks :)